python/prophet/tests/test_diagnostics.py

```diff
@@ -102,6 +102,37 @@ def mock_predict(df, model, cutoff, horizon, predict_columns):
             # check single forecast function called expected number of times
             assert n_calls == forecasts
 
+    def test_check_extra_output_columns_cross_validation(self, ts_short, monkeypatch, backend):
```

Comment:
I think copy one of the other tests -- the monkeypatch doesn't actually help us test the real logic. 
We should run the actual function, but just choose time periods such that the run is quick, since we only need to test whether the extra columns appear.